controllers/progressivesync_controller_test.go

```diff
@@ -29,60 +29,50 @@ const (
 	interval = time.Millisecond * 10
 )
 
-var _ = Describe("ProgressiveSync Controller", func() {
+var (
+	appSetAPIRef = utils.AppSetAPIGroup
```

Comment:
I tried to make this a constant but it's a pointer already so cannot be made `const`